cmd/goal/application.go

```diff
@@ -463,6 +463,9 @@ var createAppCmd = &cobra.Command{
 		// Fill in note and lease
 		tx.Note = parseNoteField(cmd)
 		tx.Lease = parseLease(cmd)
+		// DevMode fields: the default values would be ignored.
+		tx.DevMode.SkipValidation = skipValidation
+		tx.DevMode.SetNextBlockTime = setBlockTime
```

Comment:
There should be a `applyCommonFields(tx)` helper function which sets Note/Lease/DevMode